src/components/TopBar/TopBarContent.tsx

```diff
@@ -20,6 +22,22 @@ const useStyles = makeStyles((theme: Theme) => ({
   backgroundLogo: {
     background: 'url(/assets/logo.svg) no-repeat center/37px',
   },
+  breadcrumbSelected: {
+    backgroundColor: '#EF7047',
+    borderRadius: '16px',
+    color: theme.palette.TwClrBaseWhite,
+    fontFamily: 'Inter',
+    fontSize: '16px',
+    fontWeight: 500,
+    padding: '0.5em 0.8em',
+    userSelect: 'none',
+  },
+  breadcrumbSeparator: {
+    color: theme.palette.TwClrBaseGray300,
+    fontSize: '16px',
+    margin: '0 1em',
+    userSelect: 'none',
+  },
```

Comment:
Not sure if there are other values I could pull from the theme, but used the colors values I was aware of.